lib/configuration/client_configuration.js

```diff
@@ -110,38 +115,56 @@ function ConfigurationUtil(fsPromisesModule, processModule) {
    * @return {Promise<ClientConfig>} Client configuration.
    */
   this.getClientConfig = async function (configFilePath) {
+    Logger.getInstance().debug('Retrieving client config');
+
     const path = await findConfig(configFilePath);
     if (!exists(path) || path === '') {
+      Logger.getInstance().info('No config path found. Client config will not be used.');
```

Comment:
Perhaps we could explain to the user what is the fallback of that? :)

Reply:
We will describe fallback in the context, which called the function 'getClientConfig'.
I didn't want to include in this method any informations about the possible consequences in the calling context.